.eslintrc.js

```diff
@@ -24,6 +24,8 @@ module.exports = {
     'prettier',
   ],
   rules: {
+    'no-plusplus': 0,
+    'no-multiple-empty-lines': [2, {max: 3, maxEOF: 1}],
```

Comment:
Why adding this rule?

Reply:
I didn't add it in this branch, you had suggested to add this rule for removing empty lines, and I added it in the previous branch